docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_Reactive_Messsaging_SmallRye.adoc

```diff
@@ -358,6 +358,95 @@ Each of the new lines has the following meaning:
+
+* `NEVER` - This is the default value, and will be used if the mentioned resource does not exist, or if the resource exists and the attribute is not set. This turns off the MicroProfile Telemetry tracing integration, and does not allow you to turn it on in deployed applications.
+* `OFF` - Turns off the MicroProfile Telemetry tracing integration by default, but allows you to turn it on in deployed applications.
+* `ON` - Turns on the MicroProfile Telemetry tracing integration by default, but allows you to turn it off in deployed applications.
+* `ALWAYS` - Turns on the MicroProfile Telemetry tracing integration, and does not allow you to turn it off in deployed applications.
+
+Here is an example of forcing the MicroProfile Telemetry tracing integration for the AMQP connector:
+----
+/subsystem=microprofile-reactive-messaging-smallrye/opentelemetry-tracing=config:add(amqp-connector=ALWAYS)
+----
+
+This is an example of having the OpenTelemetry tracing integration off by default, but allowing deployed applications to turn it on for the Kafka connector:
+----
+/subsystem=microprofile-reactive-messaging-smallrye/opentelemetry-tracing=config:add(amqp-connector=OFF)
+----
+
+=== MicroProfile Config settings to configure tracing for deployed applications
+
+If MicroProfile Telemetry tracing is desired, as shown in the link:./MicroProfile_Telemetry.adoc#configuration[MicroProfile Telemetry subsystem] section, we *must* set the `otel.sdk.disabled` MicroProfile Config property to `true`.
+
+As mentioned in the previous section we can control tracing on the application level if the subsystem value for our connector is `OFF` or `ON`. This is done via the  MicrProfile Config properties shown below.
+
+
+These properties control tracing for an individual channel:
+** `mp.messaging.incoming.[channel-name].tracing-enabled` - Tracing setting for an outgoing channel. What type of connector this channel uses depends on the 'sibling' `mp.messaging.incoming.[channel-name].connector` property (as covered above)
+** `mp.messaging.outgoing.[channel-name].tracing-enabled` - Tracubg setting for an incoming channel. The type of connector used by the channel is determined by the value of the `mp.messaging.outgoing.[channel-name].connector` property
+
+Then we have the following properties as connector level:
+
+* `mp.messaging.connector.smallrye-amqp.tracing-enabled` - Connector level tracing setting for the AMQP connector
+* `mp.messaging.connector.smallrye-kafka.tracing-enabled` - Connector level tracing setting for the Kafka connector
+
+The resultant value of the `tracing-enabled` setting for a channel is determined as follows.
+
+First we check the lowest level property, i.e on the channel level. If that is not set, we check the value on the connector level, and if it is set we use the channel level value. The resultant value is then considered in conjunction with the value of the relevant `xxx-connector` attribute in the subsystem.
```

Comment:
Shouldn't there be `connector` instead of `channel` in `and if it is set we use the channel level value`?